src/index.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
```

Comment:
**OPTIONAL**
This file works as a global CSS style. If you feel like you do not need this, it can be deleted. If you deleted make sure to delete the import at index.js file.